test/uk/gov/hmrc/gform/sharedmodel/ExampleData.scala

```diff
@@ -298,6 +298,8 @@ trait ExampleFormTemplate {
 
   def emailTemplateId = "test-email-template-id"
 
+  def emailParameters = List(EmailParameter("fullName", "john smith"), EmailParameter("email", "test@test.com"))
```

Comment:
This suggests that the email parameter values are literal strings, but actually they need to be references to form field values, see https://confluence.tools.tax.service.gov.uk/display/DF/05.+Gforms+Specification